src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/xlf/MicrosoftNetCoreAnalyzersResources.cs.xlf

```diff
@@ -1568,7 +1568,7 @@
         <note />
       </trans-unit>
       <trans-unit id="PreferIsEmptyOverCountMessage">
-        <source>Prefer 'IsEmpty' over 'Count' to determine whether the object contains or not any items</source>
+        <source>Prefer 'IsEmpty' over 'Count' to determine whether the object is empty</source>
         <target state="translated">Při zjišťování, jestli objekt obsahuje nějaké položky, používejte spíše než Count vlastnost IsEmpty.</target>
```

Comment:
I think the status should be updated to `new` or something so that it can get translated later. Seems this update is done manually instead of generated by build. 

@BrunoJuchli you only need to update `MicrosoftNetCoreAnalyzersResources.resx` and the changes for other languages should be generated automatically with the repo build